### PR TITLE
Fix badges in the README

### PR DESCRIPTION
**Pull request by igor-makarov**
* Replace old Travis badge with GH Actions badge
* Fix the gem version shield to use HTTPS
* Remove CodeClimate as it isn't being used

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,8 @@
 
 ### CocoaPods: The Cocoa dependency manager
 
-[![Build Status](http://img.shields.io/travis/CocoaPods/CocoaPods/master.svg?style=flat)](https://travis-ci.org/CocoaPods/CocoaPods)
-[![Gem Version](http://img.shields.io/gem/v/cocoapods.svg?style=flat)](http://badge.fury.io/rb/cocoapods)
-[![Maintainability](https://api.codeclimate.com/v1/badges/8f0fe544baf2ae1acc2b/maintainability)](https://codeclimate.com/github/CocoaPods/CocoaPods/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/8f0fe544baf2ae1acc2b/test_coverage)](https://codeclimate.com/github/CocoaPods/CocoaPods/test_coverage)
```
Comment (dnkoutso) [inline README.md:8]: did e lose codeclimate as part of this?

**Review by dnkoutso — APPROVED** [review]
want to do 1-10-stable?

File: README.md (hunk 1 above)
Comment (igor-makarov) [reply README.md:8]: Not as part of this - I assumed it wasn't working because I have never seen it in PR statuses.

**Comment by igor-makarov** [comment]
What do you mean @dnkoutso? Add a badge in 1.10?

**Comment by dnkoutso** [comment]
ah nm yes! that should be ok.

**Comment by igor-makarov** [comment]
@dnkoutso can I merge this? I'm not missing anything with CodeClimate?

File: README.md (hunk 1 above)
Comment (dnkoutso) [reply README.md:8]: I thought we got it to work btw, is the link not going to be visible anymore?

Comment (igor-makarov) [reply README.md:8]: I looked at the link... frankly I'm not sure what I'm seeing.  
Looks like some sort of scoring system + another linter?  
Seems pretty hard to determine what version of the code it checked and it doesn't actually report anything to the PRs.

Who was in charge of it?

Comment (amorde) [reply README.md:8]: I recently updated this, we don't check on PRs but the badge should be up to date

Comment (amorde) [reply README.md:8]: It updates on merges to `master` I believe

**Comment by amorde** [comment]
I think we can keep CodeClimate - it doesn't have to be a check on PRs to be useful as a badge

**Pull request closed without merging**